modules/programs/rcm.nix

```diff
@@ -5,17 +5,24 @@ with lib.strings;
 let
   cfg = config.programs.rcm;
 
-  stringifyList = l:
-    if isList l then ''"${concatStringsSep " " l}"'' else stringifyList [ l ];
-
+  quotedList = l: ''"${concatStringsSep " " l}"'';
   toConfigFile = generators.toKeyValue {
-    mkKeyValue = k: v: "${toUpper k}=${stringifyList v}";
+    mkKeyValue = k: v:
+      "${toUpper k}=${if isList v then quotedList v else quotedList [ v ]}";
     listsAsDuplicateKeys = false;
   };
 in {
   options = {
     programs.rcm = {
       enable = mkEnableOption "rcm";
+      rcrc = mkOption {
+        type = types.path;
+        default = "${config.home.homeDirectory}.rcrc";
+        description = lib.mdDoc ''
+          Location of the RCRC file.
+        '';
+        example = "${home.homeDirectory}/.config/rcm/rcrc";
```

Comment:
Escape the example value, your documentation should not depend on the configuration.

Reply:
```suggestion
        example = literalExpression ''"''${home.homeDirectory}/.config/rcm/rcrc"'';
```